### DISCUSSION TITLE
Open consent screen

### DISCUSSION THREAD

**Discussion by sawilde** [orestbida/cookieconsent#620] [Q&A]
Is there a way to open the cookie consent screen from another page eg to allow a user to change their mind when on another page say the privacy policy? In v2 you could just set a role like this

```
<button type="button" data-cc="c-settings">Cookie Settings</button>
```

**Comment by orestbida** [comment]
@sawilde yes, you can do the same thing in [v3](https://cookieconsent.orestbida.com/essential/getting-started.html#how-to-open-the-preferences-modal)

**Reply by sawilde** [reply]
Can you give me hint, I had a go and nothing happened - trying to work out what I am missing.

**Reply by orestbida** [reply]
I assume you're using plain html, since you didn't specify. Are there any errors/warnings in the browsers's console tab?

You need to make sure you are actually loading the plugin on the page where you want to open the preferences modal. You can do this by checking the browser's network tab and see if it's listed there.

If it's listed, the next step is to make sure that you're loading the plugin in a deferred state, by making sure that the `type="module"` is specified in the plugin's script tag:
```html
<script type="module" src="path-to-cookieconsent-config.js"></script>
```

If the plugin is being loaded on some pages, but not on others, then it could be a path related issue. Make sure you use an absolute path in the plugin's `src` attribute.
